templates/jaasai/index.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
 
-  <div class="p-strip--image is-light" style="position:relative;background-image:url('https://assets.ubuntu.com/v1/f1511329-Background.svg') }}')">
+  <div class="p-strip--image is-light" style="position:relative;overflow:hidden;background-image:url('https://assets.ubuntu.com/v1/cf344c3f-bg.svg')">
```

Comment:
I can't find a reason for `position:relative;overflow:hidden;`?

Reply:
I think the relative is no longer required (I've removed it and it seems fine), but the `overflow:hidden` is to handle the large shadows on the bundle image (so they get cut off instead of creating horizontal scroll).